src/main/java/org/symphonyoss/integration/webhook/jira/parser/v2/JiraMetadataParser.java

```diff
@@ -233,6 +311,54 @@ protected String getLinkedIssueField(String selfPath, String key) {
   }
 
   /**
+   * This method formats the fields that support Jira's RTE formatting, changing it to the
+   * corresponding supported by Nexus.<br/>
+   * Jira's RTE syntax are not supported yet, for that reason, all jira formatting is removed in
+   * this method, and the user mentions are replaced by Nexus soft mention ([~user])
+   * @param fieldContent The field content without formatting
+   * @return Comment supported by MessageML v2 syntax
+   */
+  protected String formatTextContent(String fieldContent) {
+    String finalDescription = StringUtils.EMPTY;
+    if (StringUtils.isNotEmpty(fieldContent)) {
+      finalDescription = ParserUtils.escapeAndAddLineBreaks(fieldContent).toString();
+
+      finalDescription = JiraParserUtils.stripJiraFormatting(finalDescription);
+
+      // FIXME: Uncomment me when soft mentions are supported on MessageML v2
+//      Map<String, User> usersToMention = determineUserMentions(safeDescription.toString());
+//      if (usersToMention != null && !usersToMention.isEmpty()) {
+//        for (Map.Entry<String, User> userToMention : usersToMention.entrySet()) {
+//          User user = userToMention.getValue();
+//
+//          safeDescription.safeReplace(new SafeString(userToMention.getKey()),
+//              ParserUtils.presentationFormat(MENTION_MARKUP, user.getUsername()));
+//        }
+//      }
+
+      finalDescription = ParserUtils.markupLinks(finalDescription);
+
+    }
+    return finalDescription;
```

Comment:
You might change it to return an empty string if the parameter 'fieldContent' is empty and perform the processing otherwise.

Something like that:

```
if (StringUtils.isNotEmpty(fieldContent)) {
  String finalDescription = ParserUtils.escapeAndAddLineBreaks(fieldContent).toString();
  finalDescription = JiraParserUtils.stripJiraFormatting(finalDescription);
  return ParserUtils.markupLinks(finalDescription);
}

return StringUtils.EMPTY;
```

Using this approach you avoid instantiating a new variable when it's not required